### DISCUSSION TITLE
Development blog?

### DISCUSSION THREAD

**Discussion by Svendsys** [rust-windowing/winit#3201] [General]
Hello, I am using winit 0.2.8-something, and yesterday I tried upgrading to 0.2.9.3, which was a breaking change. It would be helpful to have some sort of info about what exactly is going to change from one version to another, with explanations, so that I don't have to read winit source code to figure that out. 

But I can't seem to find any place where winit devs talk about the versions. Understanding what's really going on, and why would be great, since winit plays a big role in the architecture of my software, and I have other dependencies that relies on integration with winit.

**Comment by notgull** [comment]
The [CHANGELOG](https://github.com/rust-windowing/winit/blob/master/CHANGELOG.md) contains version difference information.

**Reply by madsmtm** [reply]
And usually also posted under https://github.com/rust-windowing/winit/releases.

That said, I do agree that there might be a need for a more "guided" transition, maybe a short section in the front that explains the most common difficulties that people might run into when upgrading? Maybe we should be better with giving a deprecation period for things?

But then again, we're very understaffed, so probably not gonna happen.
